<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ASAP_GUI_Layout</class>
 <widget class="QMainWindow" name="ASAP_GUI_Layout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Worklistinterface</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTreeView" name="WorklistView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>256</width>
      <height>741</height>
     </rect>
    </property>
    <property name="rootIsDecorated">
     <bool>true</bool>
    </property>
    <property name="headerHidden">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTableView" name="PatientView">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>721</width>
      <height>190</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>33</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>20</number>
    </attribute>
   </widget>
   <widget class="QTableView" name="StudyView">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>270</y>
      <width>721</width>
      <height>190</height>
     </rect>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QListView" name="ImageView">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>500</y>
      <width>721</width>
      <height>281</height>
     </rect>
    </property>
    <property name="iconSize">
     <size>
      <width>200</width>
      <height>200</height>
     </size>
    </property>
    <property name="resizeMode">
     <enum>QListView::Fixed</enum>
    </property>
    <property name="viewMode">
     <enum>QListView::IconMode</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_worklists">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Worklists</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_patients">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Patients</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_studies">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>240</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Studies</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_images">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>470</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Images</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
